packages/gatsby-plugin-offline/src/gatsby-node.js

```diff
@@ -70,12 +72,11 @@ exports.onPostBuild = (args, pluginOptions) => {
     globDirectory: rootDir,
     globPatterns,
     modifyUrlPrefix: {
-      rootDir: ``,
       // If `pathPrefix` is configured by user, we should replace
       // the default prefix with `pathPrefix`.
-      "": args.pathPrefix || ``,
+      "/": args.pathPrefix || `/`,
```

Comment:
`pathPrefix` doesn't include the slash, so this should be left as it was before

Reply:
Really? In my config it does..

Reply:
https://next.gatsbyjs.org/docs/path-prefix/

Reply:
Also workbox’s examples include the slash

Reply:
There's a slash at the start but not the end, so I think the existing code is best. I'll just do some testing to check if it's working with the other changes

Reply:
That's what I mean too! For me it does not seem to be precaching files without the slash. But please do check! :)

Reply:
By the way, is it not supposed to add html files to the cache other than index and app-shell? Seeing the runtimeCaching urlPattern..